angles-java-core/src/main/java/com/github/angleshq/angles/api/requests/BaseRequests.java

```diff
@@ -20,7 +21,19 @@
 
 public abstract class BaseRequests {
 
-    protected CloseableHttpClient client = HttpClients.createDefault();
+    protected int connectionTimeout = 5000;
```

Comment:
Rather than hardcoding these timeouts, it probably extracting them into a properties file and then using maven filtering and pom properties to set defaults (that can be overridden).

Reply:
Thanks, I have update the pull request to pass the timeouts via constructor from Angles Reporter.